app/example.env

```diff
@@ -0,0 +1,22 @@
+# Auth0 related configurations
```

Comment:
Maybe add something like the following at the top of this file
`# Add your environment variables to this file and rename it to .env`

Reply:
Nit: prefer to say something along the lines of "make a copy of this file with the name .env and populate the file with your configuration". Keyword here being "copy and name" instead of "rename".